package.json

```diff
@@ -75,6 +77,7 @@
     "uuid": "^3.3.2"
   },
   "devDependencies": {
+    "@adobe/alloy": "^2.4.0-alpha.1",
```

Comment:
Are we going no need to install the latest version after every npm release or something? It's not really clear to me. You might need to update the [release steps](https://github.com/adobe/alloy/wiki/Publishing-a-New-Release).

Reply:
You are right that only full releases will need to update this. I will update the release steps.